.github/workflows/PR-validation.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       with:
         files: |
           **/*.usd*
+        files_ignore_from_source_file: .skip-usdchecker-files
```

Comment:
Nice solution, wasn't aware of it! 👍

Since the file is only used by the GitHub Actions workflow, perhaps we could move it to the `.github` folder to not pollute the root dir? We could then name it `skip-usdchecker-files.txt`.